frontend-ssr/components/exercise-submissions/Submission.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Submission, SubmissionType } from '../../redux/exercise-submissions/types';
+
+import styles from './Submission.module.scss';
+
+interface Props {
+    submission: Submission,
+}
+function ExSubmission({ submission } : Props) {
```

Comment:
Component should be named `ExerciseSubmission`. It's important to keep the same naming everywhere, his way when we search for this string we find all the files in which it's used.

Also for the skeleton, it should be `ExerciseSubmissionSkeleton`